policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/integration.yaml

```diff
@@ -163,4 +132,4 @@
     ar_personal_credits: [261]
     ar_non_refundable_credits: 261
     ar_refundable_credits: 0
-    # ar_income_tax: 687.03 expected value from TAXSIM35
+    # ar_income_tax: 687.03 #expected value from TAXSIM35
```

Comment:
should this be uncommented?

Reply:
no, this is from a previous integration test and we are still working out which model is correct, this change does not relate to this PR